extract.py

```diff
@@ -52,34 +69,68 @@ def get_plant_names(data: dict) -> dict:
     return {key: data.get(key) for key in relevant_cols}
 
 
+def get_origin_data(data: dict) -> dict:
+    """Returns origin data as dictionary."""
+
+    if 'origin_location' in data:
+        origin = data.get('origin_location')
+
+        lat, lon = origin[0], origin[1]
+        town, country_code = origin[2], origin[3]
+
+        region = origin[-1].split('/')
+        country, capital = region[0], region[1]
+
+        origin_dict = {
+            'lat': lat, 'lon': lon,
+            'town': town, 'country_code': country_code,
+            'country': country, 'capital': capital
+        }
+
+        return origin_dict
+
+    print('No origin location data.')
```

Comment:
Are the prints just so we can keep track as the pipeline runs? I think eventually we can get rid of them

Reply:
Kept them in now to keep track of progress, but yes agreed can get rid